autoload/auto_git_diff.vim

```diff
@@ -7,7 +7,7 @@ nnoremap <silent> <Plug>(auto_git_diff_manual_update) :<C-u>call auto_git_diff#s
 " Get commit hash from current line.
 " The first colomn(pick, r, ...) can be empty.
 function! s:get_git_hash() abort
-    return matchstr(getline('.'), '^\(\w\+\>\)\=\(\s*\)\zs\x\{4,40\}\>\ze')
+    return matchstr(getline('.'), '^\(\w\+\)\=\(\s\+\)\zs\x\{4,40\}\>')
```

Comment:
This pattern is based on `ftplugin/gitrebase.vim` of vim.

I think that the `\s` should be followed by a `*` to get a hash string even if a user has deleted a rebase command at the beginning of a line with a following whitespace.

If the `\s` is followed by a `*` and the `\w` is not followed by a `\>`, this function does not work correctly with a string like this:
```
410cd2e Add juce_audio_utils module
```
For this string, the function returns `cd2e`.

I think that this pattern can be '\v^(\w+\s+|\s*)\zs\x{4,40}>', but currently it is same as `ftplugin/gitrebase.vim` for compatibility.

Reply:
I understood. I think I need to remove 8c6f504

Reply:
I agree with you. Please remove the commit.